packages/dai-plugin-mcd/types/multicall/index.d.ts

```diff
@@ -0,0 +1,124 @@
+interface Currency {}
```

Comment:
can i get a brief explanation of the purpose of this, for people who aren't familiar with Typescript?

Reply:
These type definitions with JSDoc annotations allow us to provide type hints for editors like VSCode even though the implementations aren't written in TypeScript.

The type hints can work for the `WatchInterface` methods, e.g.:
<img width="674" alt="type-definitions-1" src="https://user-images.githubusercontent.com/304108/72866179-60735480-3d15-11ea-883a-0a2d775736cb.png">
And also return values (like a `Currency` object), e.g.:
<img width="791" alt="type-definitions-2" src="https://user-images.githubusercontent.com/304108/72866219-76811500-3d15-11ea-9f80-b04e5ab1f035.png">

Where the `Currency` interface is defined as:
```
interface Currency {
  /**
   * Compare two Currency values for equality by checking both the amount and currency symbol.
   * @param other The other value to compare against
   */
  isEqual(other: Currency): boolean;
}
```

Ideally the `Currency` type definitions would be located in the `@makerdao/currency` package.